config/application.rb

```diff
@@ -64,6 +64,11 @@ class Application < Rails::Application
           Sidekiq.redis { |redis| redis.ping }
           'ok'
         end
+
+        ping.check :elasticsearch do
+          Chewy.client.cluster.health["status"] == "green"
+          'ok'
```

Comment:
Hi @sahilshekhawat @tarebyte 

Sorry, I didn't get enough time to leave my feedback yesterday.

I think this will return the status of the `elasticsearch` as `ok` even it's actually `yellow` or `red` (unless there's any `exception` raised when executing `Chewy.client.cluster.health["status"]`).

I've opened #526 to address this.